src/Microsoft.Identity.Web.TokenCache/Microsoft.Identity.Web.TokenCache.csproj

```diff
@@ -11,7 +11,7 @@
     <PackageReference Include="Microsoft.Identity.Client" Version="$(MicrosoftIdentityClientVersion)" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'net6.0' Or '$(TargetFramework)' == 'net7.0' or '$(TargetFramework)' == 'net8.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net462' Or '$(TargetFrameworkIdentifier)' == 'NET6_0_OR_GREATER'">
```

Comment:
same as previous comment. careful w/taking suggestions from reviewers and not checking the code first

Reply:
Will do moving forward. Thanks Jenny!